.gitignore

```diff
@@ -0,0 +1,4 @@
+*.pyc
```

Comment:
Since these wildcards are not specific to this project, could you remove this commit? Generic wildcards like these should be added to your own `.gitignore_global`.